todo_project/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from todo import views
+
+urlpatterns = [
+    path("health/", views.HealthView.as_view()),
```

Comment:
why / after health. can you pass me full uri

Reply:
Full URI can be `http://127.0.0.1:8000/health` or `http://127.0.0.1:8000/health/` (Diff in both is the slash at the end)

Slash is added at the end (in url config) so that if URI is `http://127.0.0.1:8000/health` it redirects to `http://127.0.0.1:8000/health/`
This behaviour of redirecting is controlled by `APPEND_SLASH` config in Django with is ON by default

If we don't add a slash at the end in the url config, `http://127.0.0.1:8000/health` will work but `http://127.0.0.1:8000/health/` will give 404

Docs link - https://docs.djangoproject.com/en/5.1/ref/settings/#append-slash

Reply:
Removed the trailing slash